src/Uno.UI.Composition/Composition/Visual.cs

```diff
@@ -24,6 +24,7 @@ public partial class Visual : CompositionObject, I3DTransformableObject
 		private float _opacity = 1.0f;
 		private CompositionCompositeMode _compositeMode;
 		private ICompositionTarget? _compositionTarget;
+		private ContainerVisual? _parent;
```

Comment:
This may cause memory leaks, particularly if there are events involved. We may need to validate visuals collections.

Reply:
I'm not sure what you're referring to. `Parent` was an auto property already. I just added a backing field so that  I can add additional logic in  the setter.

Reply:
If it was there, no problem then.